src/app/services/auth/login.service.ts

```diff
@@ -57,11 +57,14 @@ export class LoginService {
     localStorage.removeItem('currentUser');
   }
   isAuthenticated(): boolean {
+    
     const user = localStorage.getItem('currentUser');
     const obj = JSON.parse(user);
-    if (obj){
-      console.log(this.jwtHelper.getTokenExpirationDate(obj.token));
-      return !this.jwtHelper.isTokenExpired(obj.token);
+    if (obj!=null){
+      if (!this.jwtHelper.isTokenExpired(obj.token)){
```

Comment:
@jmilleriec you can skip all this and just 

return (!this.jwtHelper.isTokenExpired(obj.token))